nautobot/extras/api/views.py

```diff
@@ -441,9 +441,18 @@ def sync(self, request, pk):
             raise CeleryWorkerNotRunningException()
 
         repository = get_object_or_404(GitRepository, id=pk)
-        repository.sync(user=request.user)
-        return Response({"message": f"Repository {repository} sync job added to queue."})
 
+        job_result = repository.sync(user=request.user)
+
+        return Response({
+                "message": f"Repository {repository} sync job added to queue.",
+                "repository_id": repository.id,
+                "repository_name": repository.name,
+                "sync_jobmodel_id": job_result.job_model_id,
```

Comment:
The Job ID probably isn't needed here since it will be the same for all GitRepository syncs.

Reply:
My thinking here was to allow an easy lookup of the `queue` as per the original enhancement request (which I did write, to be fair!). If we drop this, it's two queries to find out what queue we're scheduled on, rather than just one.

Happy to drop it though!